### ISSUE TITLE
[System UI Controller] Preview is not rendering

### ISSUE THREAD

**Issue by rewgoes** [google/accompanist#918]
Preview started throwing an exception after commit bb866bf3a1e96eb14a746649dcc9bb39878542af

```
java.lang.IllegalArgumentException: The Compose View must be hosted in an Activity with a Window!  at
com.google.accompanist.systemuicontroller.AndroidSystemUiController.<init>(SystemUiController.kt:177)   at
com.google.accompanist.systemuicontroller.SystemUiControllerKt.rememberSystemUiController(SystemUiController.kt:165)
```

https://github.com/google/accompanist/blob/ce11b260f2d2e5d69d515833b4437cdb2c67e82d/systemuicontroller/src/main/java/com/google/accompanist/systemuicontroller/SystemUiController.kt#L177

**Comment by AKJAW** [comment]
The same happened to us after updating Kotlin to 1.6.10. Our theme uses SystemUiController and all of the previews are broken. 

Our current work around is a boolean flag which disables SystemUiController on previews
```kotlin
@Composable
fun Theme(
    themeMode: AppThemeMode,
    isPreview: Boolean = true,
    content: @Composable () -> Unit = {},
) {
    
    if (isPreview.not()) {
       //  SystemUiController stuff
    }

    // ...
}
```

Our compose versions:
``` groovy
version.androidx.compose.compiler=1.1.0-rc02
version.androidx.compose.foundation=1.1.0-rc01
version.androidx.compose.material=1.1.0-rc01
version.androidx.compose.ui=1.1.0-rc01
version.google.accompanist=0.22.0-rc
```

**Comment by abdulelahragih** [comment]
Same here after updating to Kotlin 1.6.10

**Comment by yue-li-qm** [comment]
still broken

**Comment by alexwhb** [comment]
I'm running into this issue as well. Any timeline on when this'll be released?

**Comment by alexvanyo** [comment]
This should be fixed in the latest alpha release `0.24.3-alpha`.
